app/views/schools/participants/edit_email.html.erb

```diff
@@ -4,10 +4,17 @@
 
     <span class="govuk-caption-xl"><%= @school.name %></span>
 
-    <%= form_for @induction_record.preferred_identity, url: schools_participant_update_email_path, method: :put do |f| %>
-    <%= f.govuk_error_summary %>
-
-      <%= f.govuk_text_field :email, label: { text: "Change #{@profile.ect? ? "ECT’s" : "mentor’s"} email address", tag: 'h1', size: 'xl' } %>
+    <%= form_with model: @induction_record.preferred_identity,
+                  scope: '',
+                  url: schools_participant_update_email_path,
+                  method: :put do |f| %>
+      <%= f.govuk_error_summary %>
+      <%= f.govuk_text_field :email,
+                             value: nil,
+                             class: 'govuk-!-width-two-thirds',
+                             hint: -> { tag.p(t('participant.edit_email_hint')) },
+                             label: -> { tag.h1(t('participant.edit_email', name: @profile.full_name),
+                                                class: 'govuk-heading-xl') } %>
```

Comment:
I don't think you need the proc here:

```suggestion
                             label: { text: t('participant.edit_email', name: @profile.full_name), size: 'xl', tag: 'h1' }
```
```

Reply:
Initially I wrote it like your suggestion. 
However, that left a very short space between the label and the input which should be much bigger as per design.
Using a proc is the only way I could figure out to achieve it. 

Now that you say that, I have checked and it is the css class what I missed, so I have tried to add it to your approach and it works!. Thanks I will change it!